src/common/storage.js

```diff
@@ -367,6 +368,13 @@ class RNStorage {
   }
 
   getStorageVersion = async () => this.load({ key: STORAGE_VERSION })
+
+  getLatestVersion = async () => {
+    const latestVersion = await this.load({ key: LATEST_VERSION });
```

Comment:
The app doesn't know what version itself is, and it has to read from storage?

This looks strange to me. @AmazingBeerBelly do you know how to properly read version number of the app?